src/services/sessions/models/session.model.ts

```diff
@@ -0,0 +1,55 @@
+import mongoose, { Schema, Document } from 'mongoose';
+
+// Could probably be abstracted into a booking and used for workshop bookings as well
+export interface ISession extends Document {
+    date: string;  // Could probs remove this
+    startTime: string; // Should be a date object
+    endTime: string; // Should be a date object
+    room: string;
+    advisor: string;
+    type: string;
+    currentBooking: ISessionDetails;
+    waitingList: Array<ISessionDetails>;
+    isCurrentBooking: boolean;
+    // attendedNotAttended: string; (not sure what this is)
+}
+
+interface ISessionDetails {
+    studentId?: string;
+    reason: string // Taken from 'this appointment is for'
+    subjectName: string;
+    assignmentType: string;
+    isGroupAssignment: boolean;
+    needsHelpWithOptions: IOption;
+    additionalHelpDetails: string;
+    additionalOptions?: IOption;
+}
+
+export interface IOption {
+    key: string;
```

Comment:
This should probably be name something a little better or have a comment for what it does